assets/js/view/spreadsheet.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+module View
+  class Spreadsheet < Snabberb::Component
+    needs :game
+
+    def render
+      props = {
+         style: {
```

Comment:
have you tested this on mobile? my guess is that it won't scroll properly, look at map / stock market for guidance on how to make thi mobile friendly

Reply:
ah, I see what you mean by scrolling properly. Ok, I will add support for that